src/Dapr.Actors/Runtime/ActorTimerManager.cs

```diff
@@ -27,6 +27,13 @@ public abstract class ActorTimerManager
         /// <returns>A task which will complete when the operation completes.</returns>
         public abstract Task RegisterReminderAsync(ActorReminder reminder);
 
+        /// <summary>
+        /// Gets a reminder previously registered using
+        /// </summary>
+        /// <param name="reminder">The <see cref="ActorReminderToken" /> to unregister.</param>
+        /// <returns>A task which will complete when the operation completes.</returns>
+        public abstract Task<IActorReminder> GetReminderAsync(ActorReminderToken reminder);
```

Comment:
The underlying `IDaprInteractor` allows for cancellation, but the higher-level types don't seem to make use of it.  Should we start doing so for new APIs?  It'd likely be breaking to do so as a whole, but one has to start somewhere.